misc/klayout/build.sh

```diff
@@ -7,18 +7,34 @@ set -x
 UNAME_OUT="$(uname -s)"
 case "${UNAME_OUT}" in
     Linux*)     OS=Linux;;
+    Darwin*)    OS=Mac;;
     *)          OS="${UNAME_OUT}"
                 echo "Unknown OS: ${OS}"
                 exit;;
 esac
 
 cd ${SRC_DIR}
-./build.sh -build "${SRC_DIR}/build" -python "${PYTHON}" -expert -without-qtbinding -libpng -libexpat -dry-run
+if [[ $OS == "Linux" ]]; then
+    ./build.sh -build "${SRC_DIR}/build" -python "${PYTHON}" -expert -without-qtbinding -libpng -libexpat -dry-run
+    bin_ext=""
+    lib_ext=".so"
+elif [[ $OS == "Mac" ]]; then
+    # Support `any_cast` workaround: https://conda-forge.org/docs/maintainer/knowledge_base.html#newer-c-features-with-old-sdk
+    export CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
+    ./build.sh -build "${SRC_DIR}/build" -python "${PYTHON}" -expert -without-qtbinding -libpng -libexpat -dry-run
+    bin_ext=".app"
+    lib_ext=".dylib"
+fi
 
 cd ${SRC_DIR}/build
 make V=1 -j$CPU_COUNT
 make V=1 install
 
 cd ${SRC_DIR}/bin-release
-cp -a klayout strm* ${PREFIX}/bin/
-cp -ar *.so* pymod *_plugins ${PREFIX}/lib/
+cp -a klayout${bin_ext} strm* ${PREFIX}/bin/
+cp -a *${lib_ext}* pymod *_plugins ${PREFIX}/lib/
+
+if [[ $OS == "Mac" ]]; then
+    cd ${SRC_DIR}
```

Comment:
Maybe we could patch the Makefile to do this?

Reply:
It seems like it should do this as part of the install, but it doesn't, and I'm not sure why not.